tests/instancetype_test.go

```diff
@@ -225,9 +221,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		})
 
 		It("[test_id:TODO] should apply instancetype and preferences to VMI", func() {
-			vmi := tests.NewRandomVMIWithEphemeralDisk(
-				cd.ContainerDiskFor(cd.ContainerDiskCirros),
-			)
+			vmi := libvmi.NewCirros()
```

Comment:
https://github.com/kubevirt/kubevirt/pull/8280 ^_^ I honestly don't mind if you just amend that entire commit in here tbh. Not all of the tests using the disk actually required it FWIW.

Reply:
Ok, I will do it.